CHANGELOG.md

```diff
@@ -1,3 +1,15 @@
+# dbt_tiktok_ads v0.2.0
+- This PR ([#5](https://github.com/fivetran/dbt_tiktok_ads/pull/5)) applies the Ad Reporting V2 updates to provide further flexibility in your ad reporting. Additionally, these new end models will be leveraged in the respective downstream [dbt_ad_reporting](https://github.com/fivetran/dbt_ad_reporting) models. 
+
+- Adds in the `account_report`, `url_report`, `ad_report` models and the respective field descriptions and tests for the new models in the yml files
+- Adds additional fields to existing models `ad_group_report` and `campaign_report` that were previously not brought in
+- Removes the ad_adapter model and the respective dependencies the other models have on it, instead using that model's level of granularity's hourly report.
+- Add passthrough metrics for respective models: ([#5](https://github.com/fivetran/dbt_tiktok_ads/pull/5))
```

Comment:
I think if you decide to reference the PR once above, you can probably remove from here!